### ISSUE TITLE
[Bug] AF does not remember advanced settings

### ISSUE THREAD

**Issue by sakej** [Marekkon5/onetagger#92]
Hi.

1. Open OT
2. Go to AF and change any Advanced settings
3. Go to AT
4. Go back to AF
Setting are reverted

Build #203 (portable version)
Win 10

**Comment by sakej** [comment]
True, but this is different behavior than every else in application where settings are saved instantly

**Comment by Marekkon5** [comment]
I've fixed this in the latest commit, you can get the build from the Actions tab. Feel free to comment/reopen if you have any more issues. Thanks
